cli-core/reportGlobal.js

```diff
@@ -110,7 +110,13 @@ async function create_global_report(reports,options){
     }
     //Add info the recap sheet
     //Prepare data
-    const isMobile = (await axios.get('http://ip-api.com/json/?fields=mobile',{proxy})).data.mobile //get connection type
+    let isMobile = (DEVICE !== "desktop");
+    let ipApiError = null;
+    try {
+        isMobile = (await axios.get('http://ip-api.com/json/?fields=mobile',{proxy})).data.mobile //get connection type
```

Comment:
`isMobile` could possibly be assigned to `null` in case of a weird OK response with an empty or unexpected response body. Would it be relevant to assign a default value here too? e.g. `isMobile = (...)?.data?.mobile ?? isMobile`?

Reply:
Maybe we should take advantage of this PR to also modify the URL to use the `https` one.

Reply:
The https URL isn't free to use

Here the pricing section : https://members.ip-api.com/#pricing (I just knew it thanks to https://ip-api.com/json/)